### DISCUSSION TITLE
ADTS supported languages

### DISCUSSION THREAD

**Discussion by traugutt** [deepgram/community#187] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

Hello! We're getting empty responses for Hebrew, Turkish and French audio files in adts format, but Spanish is processed fine, albeit the transcription is a bit inaccurate. Is there any known limitation for certain languages? I see that the extension is not on the list of supported extensions. Is it just a coincidence that Spanish adts file is processed in some capacity? Any guidance is really appreciated, thank you.

### If possible, please attach your code or paste it into the text box.

def deepgram_transcribe(url, params, input_type):
    source = {"url": url}
    response = deepgram.transcription.sync_prerecorded(
        source,
        {"punctuate": True,
         "language": params.language,
         "diarize": True if input_type == TextType.conversation else False,
         "tier": "enhanced"},
    )
    words = response["results"]["channels"][0]["alternatives"][0]["words"]
    transcript = response["results"]["channels"][0]["alternatives"][0]["transcript"]
    if not bool(words) and not bool(transcript):
        logger.warning(f'Deepgram returned empty result')
        raise ApiException(error_code=ErrorCodes.AudioTranscribeFailed)

**Comment by jjmaldonis** [comment, marked as answer]
Hey @traugutt, you are right that we do not technically suppport adts format. If it's working for Spanish that's great, and it will likely work for other languages.

The reason you are getting empty responses is likely because the `enhanced` tier does not support all languages (see https://developers.deepgram.com/docs/language). Hebrew is not supported, and Turkish is only available for the `base` tier. French is currently in Beta for the `enhanced` tier, so that may be why it isn't working at the moment. If you use `tier=base` then you will have additional language support.
